weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/ecpTypes.ts

```diff
@@ -42,6 +42,8 @@ export type EvaluationComparisonResults = {
   resultRows: {
     // Each rowDigest is a unique identifier for the input data.
     [rowDigest: string]: {
+      // For imperative evaluations, we store the original input directly
+      originalInput?: any;
```

Comment:
I think this is fragile to say: "for imperative evaluations". The reason is if the data is simple, you are most definitely going to get digest collisions (hopefully - meaning you can compare evaluations from two types). I think you should call this `rawDataRow` and just say that this is an optional cache of the raw data row which is more frequently populated if one of the evals was run an in imperative way